core/src/main/java/cucumber/api/formatter/StrictAware.java

```diff
@@ -1,9 +1,11 @@
 package cucumber.api.formatter;
 
+import cucumber.api.Plugin;
+
 /**
- * Interface for Formatters that need to know if the Runtime is strict.
+ * Interface for Plugins that need to know if the Runtime is strict.
  */
-public interface StrictAware extends Formatter {
+public interface StrictAware extends Plugin {
```

Comment:
Do you think it would be a good idea for all marker interfaces to extend from Plugin?

Reply:
Not sure what you mean; did I miss anything?

Reply:
The EventListner I believe

Reply:
EventListener is not a marker interface; it has a method: `void setEventPublisher(EventPublisher publisher);`

Reply:
Good one. 

Would it be  a good idea to make EventListener extend from Plugin?

Reply:
Yes. (Updated)